### DISCUSSION TITLE
Quick actions on app icon (iOS)

### DISCUSSION THREAD

**Discussion by siticum** [buildnatively/community#79] [Ideas]
### Use case

```Text
Hi, would would love to add a series of quick actions for our app. Any chance this is possible to integrate in the natively wrapper?

Use case, "Start a tour" and the app opens and starts a workflow.
```

### Idea

```Text
Define quick actions in an Natively element and have them trigger a workflow and series of actions. Similar to deeplinks.
```

**Comment by siticum** [comment]
Example: 
![427821980_2100883560275719_9091345146551161353_n](https://github.com/buildnatively/community/assets/151831088/8e439534-12d8-431b-ae75-95b35b55fc5d)

**Reply by romanfurman6** [reply]
That actually a great suggestion! Thanks!

**Reply by romanfurman6** [reply]
It should be with some shortcut URLs, correct?

**Reply by siticum** [reply]
Yes - URL deeplinks, or even better - A trigger for a workflow. That would make the hassle of setting up URLs and deeplings completely redundant.
